src/js/editor/editor.js

```diff
@@ -307,20 +299,27 @@ class Editor {
 
   handleDeletion(event) {
     event.preventDefault();
+    const key = Key.fromEvent(event),
+          direction = key.direction;
 
     let offsets = this.cursor.offsets;
     let currentMarker, currentOffset;
-    this.run((postEditor) => {
+
+    this.run(postEditor => {
       let results;
       if (this.cursor.hasSelection()) {
         results = postEditor.deleteRange(offsets);
       } else {
-        // FIXME: perhaps this should accept this.cursor.offsets?
-        results = postEditor.deleteCharAt(offsets.headMarker, offsets.headOffset-1);
+        const deletePosition = {
+          marker: offsets.headMarker,
+          offset: offsets.headOffset
+        };
+        results = postEditor.deleteFrom(deletePosition, direction);
```

Comment:
I think when we have a lot of potentially different input formats floating around we're likely missing an abstraction. I've been wanting a `Position` or `CursorPosition` container that makes passing around positional information easier. The `deleteFrom({marker, offset})` could become `deleteFrom(positionInstance)` in the future.

Having a position abstraction would open up `deleteRange` so that we could do things like `deleteRange(from=cursorPosition, to=cursorPosition.moveBackOne())` and `deleteRange(from=cursorPosition, to=cursorPosition.moveForwardOne())`. I would like to tackle that in a future PR, though. There is good test coverage for `deleteFrom` now that we can lean on until then.